### ISSUE TITLE
[🐛 Bug]: Sometimes noVNC doesn't work

### ISSUE THREAD

**Issue by Doofus100500** [SeleniumHQ/docker-selenium#2091]
### What happened?

Sometimes noVNC doesn't work, even though "se:vncEnabled": true is present in the capabilities

To reproduce this you can use my test
[SeleniumTest.zip](https://github.com/SeleniumHQ/docker-selenium/files/13962273/SeleniumTest.zip)

![image_2024-01-17_13-44-12](https://github.com/SeleniumHQ/docker-selenium/assets/86063046/52a73b92-ff76-43b0-81cc-530afa6e3ac2)
![image_2024-01-17_13-44-38](https://github.com/SeleniumHQ/docker-selenium/assets/86063046/cb5ec074-ec6d-4576-932b-4deea86d88fe)

### Command used to start Selenium Grid with Docker (or Kubernetes)

```shell
helm
```

### Relevant log output

```shell
Capabilities:

{
  "acceptInsecureCerts": false,
  "browserName": "firefox",
  "browserVersion": "120.0.1",
  "grid:teamName": "idevops",
  "moz:accessibilityChecks": false,
  "moz:buildID": "20231129155202",
  "moz:geckodriverVersion": "0.33.0",
  "moz:headless": false,
  "moz:platformVersion": "5.4.0-164-generic",
  "moz:processID": 144,
  "moz:profile": "/tmp/rust_mozprofileZOyeGo",
  "moz:shutdownTimeout": 60000,
  "moz:webdriverClick": true,
  "moz:windowless": false,
  "pageLoadStrategy": "normal",
  "platformName": "linux",
  "proxy": {},
  "se:noVncPort": 7900,
  "se:vncEnabled": true,
  "se:vncLocalAddress": "ws://10.223.126.118:7900",
  "setWindowRect": true,
  "strictFileInteractability": false,
  "timeouts": {
    "implicit": 0,
    "pageLoad": 300000,
    "script": 30000
  },
  "unhandledPromptBehavior": "dismiss and notify"
}`
```

### Operating System

k8s

### Docker Selenium version (image tag)

4.16.1-20231219

### Selenium Grid chart version (chart version)

0.26.4

**Comment by Doofus100500** [comment]
Add the test to reproduce this

**Comment by VietND96** [comment]
Can you take a look any similar to this? https://github.com/SeleniumHQ/docker-selenium/issues/2084

**Comment by VietND96** [comment]
Via screenshot I understand that you want to point a session without recorder button is visible? It looks like Xvfb not started or crashed in that node

**Comment by Doofus100500** [comment]
> It looks like Xvfb not started or crashed in that node

No, the logs are fine on this and even the video of the session successfully recorded on the s3

**Comment by VietND96** [comment]
There was a fix for delay in x11vnc (#2119). Can you use `nightly` build to recheck and confirm?

**Comment by VietND96** [comment]
In addition, there was a fix for issue where `xauth` not found, preventing xvfb from starting - https://github.com/SeleniumHQ/docker-selenium/commit/a9dc7b760d225dfce2bc8adce8b6d7e1c491278f
Please use `nightly` build to recheck and confirm.
I will mark this as completed. Let's reopen if the intermittent issue occurs again. Thanks!
